build/download_kubectl_binary.sh

```diff
@@ -5,22 +5,6 @@ download_kubectl_binary(){
   local KUBECTL_VERSION=$3
   local KUBECTL_BIN_NAME="kubectl"
 
-  if [ "$PLATFORM" = "win" ]; then
-    PLATFORM="windows"
-  fi
-
-  if [ "$ARCH" = "armhf" ]; then
-    ARCH="arm"
-  fi
-
-  if [ "$ARCH" = "aarch64" ]; then
-    ARCH="arm64"
-  fi
-
-  if [ "$ARCH" = "x86_64" ]; then
-    ARCH="amd64"
-  fi
```

Comment:
PLATFORM and ARCH now come from `go env` output and they are already in arm, arm64 or windows output.